src/swap/events.ts

```diff
@@ -6,6 +6,9 @@ export enum SwapEventName {
   SWAP_DETAILS_EXPANDED = 'Swap Details Expanded',
   SWAP_ERROR = 'Swap Error',
   SWAP_ESTIMATE_GAS_CALL_FAILED = 'Swap Estimate Gas Call Failed',
+  SWAP_FIRST_INPUT = 'Swap First Input',
+  SWAP_FIRST_QUOTE_FETCH = 'Swap First Quote Fetch',
```

Comment:
Do we log all swap quotes received? If so, I think it'd be good to go with the alternative in this case as proposed in the doc for consistency on quote metrics

Reply:
yeah we do - i'm open to it